index.d.ts

```diff
@@ -1,313 +0,0 @@
-// Type definitions for prosemirror-tables 0.8
-// Project: https://github.com/ProseMirror/prosemirror-tables
-// Definitions by: Oscar Wallhult <https://github.com/superchu>
-//                 Eduard Shvedai <https://github.com/eshvedai>
-//                 Patrick Simmelbauer <https://github.com/patsimm>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
-import {
-  EditorState,
-  Plugin,
-  SelectionRange,
-  Transaction,
-  PluginKey,
-  Selection,
-} from 'prosemirror-state';
-import {
-  Node as ProsemirrorNode,
-  NodeSpec,
-  Slice,
-  ResolvedPos,
-  Schema,
-  NodeType,
-} from 'prosemirror-model';
-import { Mappable } from 'prosemirror-transform';
-import { EditorView, NodeView } from 'prosemirror-view';
-
-export interface TableEditingOptions {
-  allowTableNodeSelection?: boolean;
-}
-
-export interface TableNodesOptions {
-  tableGroup?: string;
-  cellContent: string;
-  cellAttributes: { [key: string]: CellAttributes };
-}
-
-export type getFromDOM = (dom: Element) => any;
-export type setDOMAttr = (value: any, attrs: any) => any;
```

Comment:
I guess maybe we should still export these two types, to avoid breaking changes.

Reply:
done, albeit with slightly stricter types